### PR TITLE
Tabs Atom

### PR DESCRIPTION
**Pull request by 0c370t**

### CODE REVIEW

File: clients/web/src/lib/components/atoms/Tabs/types.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import type {SvelteComponent, ComponentProps} from "svelte";
+import type {IconSource} from "@steeze-ui/svelte-icon/types";
+
+// export type TabItem = {
```
Comment (zachstence) [inline clients/web/src/lib/components/atoms/Tabs/types.ts:4]: Remove comment

File: clients/web/src/lib/components/atoms/Tabs/types.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export type TabItem<T extends SvelteComponent = SvelteComponent> = {
+    component: T;
+    props: ComponentProps<T>;
+    icon?: IconSource;
+};
+
+export type TabItems = {
+    [key: string]: TabItem;
+};
```
Comment (zachstence) [inline clients/web/src/lib/components/atoms/Tabs/types.ts:18]: Can be simplified

```suggestion
export type TabItems = Record<string, TabItem>;
```

File: clients/web/src/lib/components/atoms/Tabs/Tabs.svelte [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    interface $$Props {
+        tabs: TabItems;
+        activeKey: keyof TabItems;
+    }
+
+    export let tabs: TabItems;
+    export let activeKey: keyof TabItems;
+</script>
+
+<div class="text-sm font-medium text-center border-b text-gray-400 border-gray-700">
```
Comment (zachstence) [inline clients/web/src/lib/components/atoms/Tabs/Tabs.svelte:15]: There are some accessibility considerations we need to implement

See
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role

For implementation, the official `react-tabs` library would be good to follow (there are several `aria` properties we should implement)

**Review by zachstence — CHANGES_REQUESTED** [review]

File: clients/web/src/lib/components/atoms/Tabs/types.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import type {SvelteComponent, ComponentProps} from "svelte";
+import type {IconSource} from "@steeze-ui/svelte-icon/types";
+
+// export type TabItem = {
+//   component: SvelteComponent,
+//   key: string,
+//   props: ComponentProps<SvelteComponent>
+// }
+
```
Comment (0c370t) [inline clients/web/src/lib/components/atoms/Tabs/types.ts:9]:
```suggestion
```

**Review by zachstence — APPROVED** [review]

**Pull request merged**